frontend/packages/kubevirt-plugin/integration-tests/tests/vm.cloud-init.scenario.ts

```diff
@@ -0,0 +1,62 @@
+import { testName } from '../../../../integration-tests/protractor.conf';
+import { withResource } from '../../../console-shared/src/test-utils/utils';
```

Comment:
And '@console/shared/src/test-utils/utils'

Reply:
done